src/components/icons/ImageIcon.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+export function ImageIcon({ fill = '#4A5568', ...props }) {
```

Comment:
Are we holding onto this, beyond an example? I like the style, though I think the naming of this component could be more explicit, it sounds very generic.